system-linux.c

```diff
@@ -2545,8 +2573,8 @@ ethtool_feature_value(const char *ifname, const char *keyname)
 	return active;
 }
 
-int
-system_if_dump_info(struct device *dev, struct blob_buf *b)
+static int
+system_if_dump_info_legacy(struct device *dev, struct blob_buf *b)
```

Comment:
If we touch this function declaration anyway we can as well make it `static void`, it can only ever return `0` (which is subsequently ignored by the caller)